config/kubernetes/sig-api-machinery/teams.yaml

```diff
@@ -16,6 +16,22 @@ teams:
     - lavalamp
     - liggitt
     privacy: closed
+  sig-api-machinery-members:
+    description: Members of sig-api-machinery
+    members:
```

Comment:
So there is more people that is regular member, how do we get the list populated? we announce this change and ask them to add themselves via PR?

Reply:
Oh, check the API machinery agenda doc. @lavalamp already intends to announce this, which is why I decided to do this.